perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -2405,59 +2692,59 @@ private void removeFormerMembersWhileSynchronization(PerunSession sess, Group gr
 				if(authoritativeGroupValue == 1) thisGroupIsAuthoritativeGroup = true;
 			}
 		} catch (AttributeNotExistsException ex) {
-			//Means that this group is not authoritative
+			// Means that this group is not authoritative
 			log.error("Attribute {} doesn't exists.", A_G_D_AUTHORITATIVE_GROUP);
 		}
 
-		//Second remove members (use authoritative group where is needed)
+		// Second remove members (use authoritative group where is needed)
 		for (RichMember member: membersToRemove) {
-			// Member is missing in the external group, so remove him from the perun group
+			// Member is missing in the external group, so remove him from the Perun group
 			try {
-				//members group
+				// Members group
 				if (group.getName().equals(VosManager.MEMBERS_GROUP)) {
 					// If the group is members group, the member must be disabled as a member of VO
 					try {
 						getPerunBl().getMembersManagerBl().disableMember(sess, member);
 						log.info("Group synchronization {}: Member id {} disabled.", group, member.getId());
 					} catch(MemberNotValidYetException ex) {
-						//Member is still invalid in perun. We can delete him.
+						// Member is still invalid in perun. We can delete him.
```

Comment:
I did it because of bachelor's thesis. I needed to have most of comments in code, which was part of my thesis, written with same convention. And I think its better to write this type of comments like // Text instead of //text.